plugin/src/main/java/org/opensearch/ml/processor/MLInferenceSearchRequestProcessor.java

```diff
@@ -251,7 +251,7 @@ public void onResponse(Map<Integer, MLOutput> multipleMLOutputs) {
                             requestListener.onResponse(request);
                         }
                     } catch (Exception e) {
-                        if (ignoreMissing || ignoreFailure) {
+                        if (ignoreFailure) {
```

Comment:
Change to `isIgnoreFailure` to clearly shows its a boolean variable?

Reply:
ignoreFailure is variable name inherited from search processors, and it's widely used variable in all types of processors. Keeping it for consistency.

Reply:
Also for the `ignoreMissing` variable, if it still exists

Reply:
Ok fine